docs/blog/2019-02-06-government-open-data-site-with-gatsby/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "How we're migrating a government open data site to Gatsby"
+date: 2019-02-05
+authors:
+  - Ryan Johnson
+  - Jeff Keene
+tags:
+  - open data
+  - government
+  - design
+  - data
+  - d3
+  - case-studies
+canonicalLink: https://revenuedata.doi.gov/blog/homepage-revamp-part-two/
+---
+
+_This post was originally published at [Open data design at the U.S. Department of the Interior](https://revenuedata.doi.gov/blog/)._
+
+A few months ago, our small team at the Office of Natural Resources Revenue (ONRR) decided we needed to refactor our [open-data website](https://revenuedata.doi.gov/). We knew we were in for a massive undertaking, as the site was originally built with the static-site generator [Jekyll](https://jekyllrb.com/), with significant custom coding to furnish bespoke features, automate data updates, and compile and deploy the site with [18F's](https://18f.gsa.gov/) fantastic static-site hosting service, [Federalist](https://federalist.18f.gov/).
+
+As covered in our [first post in this series](/homepage-revamp/), we were simultaneously scoping a redesign of our homepage. Rebuilding the homepage would allow us to test some of our assumptions about how to address obstacles in our existing workflow.
```

Comment:
You'll have to make these links absolute (e.g. prepend the original blog post URL).

Reply:
Ooops! Yes, thank you.